### DISCUSSION TITLE
Test fix GH-193

### DISCUSSION THREAD

**Discussion by gand07** [Daveiano/weewx-wdc#210] [Q&A] [answered]
Hi
If I want to test this fix, which file should I download?

Gert

**Comment by Daveiano** [comment]
Replace the file at `dist/main.css` with the attached [file](https://github.com/Daveiano/weewx-wdc/files/13115338/main.zip) (de-compress first).

**Reply by gand07** [reply]
Hi
Thanks for your support.
I have installed main.js and main.css as you proposed in order to solve the Tool tip issue (GH-194) and the make the sidebar scrollable (GH-193)

I can't see any changes efter installing the new files files. Should I delete any files in the public_html directory? Furthermore it seems that  the min/max date/time are missing for today, yesterday week and month. Year is fine

This also applies to your demo page

Gert

**Reply by Daveiano** [reply]
Did you copy the files to the `weewx-wdc/dist` folder inside the [`SKIN_ROOT`](https://weewx.com/docs/usersguide.htm#Where_to_find_things)?

Yes, please try to remove at least the whole `dist` folder inside of the `public_html` directory and wait for the next report run (or trigger it).

Otherwise, it could also be a cache issue, did you try to access the page in an incognito browser window?

> Furthermore it seems that the min/max date/time are missing for today, yesterday week and month. Year is fine

That's the default, as configured in skin.conf (for a year, it is always shown): https://github.com/Daveiano/weewx-wdc/blob/3.x/skins/weewx-wdc/skin.conf#L132

**Comment by gand07** [comment, marked as answer]
It's working now with GH-193 and GH-194, thanks for that.
I copied the main.js and main.css to the weewx-wdc/dist folder, but no changes. I then looked in the public_html/dist folder and the old version of the 2 files were here. I deleted the public_html/dist folder, but no new files were generated. I used my backup for that folder and copied the 2 new files into the directory..
I don't know why the public_html/dist  was not generated., but it's working now.

Thanks for pointing me in the right direction.

Gert
